File: PHP/uberVan.php
<?php 
require_once('car.php');
class UberVan extends Car {
    private $typeCarAccepted;
    private $seatMaterial;

    //Constructor1
    public function __construct(String $license, Account $driver, $typeCarAccepted = null, $seatMaterial = null) {
        parent::__construct($license, $driver);
        $this->typeCarAccepted = $typeCarAccepted ?? [];
        $this->seatMaterial = $seatMaterial ?? [];
    }

    //Metodo sobreescribir Polimorfismo
    public function setPassenger($passenger) {
        if ($passenger == 6) {
            $this->passenger = $passenger;
        } else {
            echo "Necesitas asignar 6 pasajeros!";
        }
    }
}
?>

File: C#/UberVan.cs
using System;

namespace UberPoo
{
    public class UberVan : Car
    {
        Dictionary<string, Dictionary<string, int>> typeCarAccepted = new Dictionary<string, Dictionary<string, int>>();
        List<string> seatMaterial = new List<string>();

        public UberVan(string license, Account driver)
            : base(license, driver)
        {
        }

        public UberVan(string license, Account driver, Dictionary<string,
            Dictionary<string, int>> typeCarAccepted, List<string> seatMaterial)
            : base(license, driver)
        {
            this.typeCarAccepted = typeCarAccepted;
            this.seatMaterial = seatMaterial;
        }

        public new void SetPassenger(int passenger)
        {
            if (passenger == 6)
            {
                this.passenger = passenger;
            }
            else
            {
                Console.WriteLine("Necesitas asignar 6 pasajeros!");
            }
        }
    }
}
